src/components/Board.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { Grid, Paper } from "@mui/material";
+
+const formatNumber: (num: number) => string = (num: number) =>
+  num === 0 ? "_" : num < 10 ? `0${num}` : `${num}`;
+
+const Board: React.FC = () => {
+  const arr: Array<number> = React.useMemo(
+    () => Array.from({ length: 90 }, (_) => 0),
```

Comment:
Add config file and then give value to length from that env variable

Reply:
completed